tests/python/test_memory.py

```diff
@@ -45,5 +45,5 @@ def get_process_memory():
         X().run()
         gc.collect()
         curr_mem = get_process_memory()
-        assert (curr_mem - ref_mem < 1E-1
-                )  # shouldn't increase more than 0.1 MB each loop
+        assert (curr_mem - ref_mem < 6E-1
```

Comment:
In https://github.com/taichi-dev/taichi/actions/runs/3467496526/jobs/5792421314, we can see the difference is larger than 1.2:
```
assert (282.353664 - 281.100288) < 0.1
```

Reply:
agh this one is indeed flaky...